specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -0,0 +1,13520 @@
+                        "locale": "en-US",
+                        "name": "Base model",
+                        "description": "Base model description",
+                        "properties": {}
+                      },
+                      "datasets": [
+                        {
+                          "dataImportKind": "Language",
+                          "id": "c93d0931-ba7f-4ca5-b2a9-11fe4bdcf16d",
+                          "createdDateTime": "2019-03-20T14:46:18Z",
+                          "lastActionDateTime": "2019-03-20T14:46:18Z",
+                          "status": "Succeeded",
+                          "locale": "en-US",
+                          "name": "Language dataset",
+                          "properties": {
+                            "AcceptedLines": "11",
+                            "RejectedLines": "2",
+                            "ReportUri": "https://www.example.org/LanguageData/e75c10b6-f6c3-4693-9fc7-a6baf32b599f?st=2018-02-09T18%3A07%3A00Z&se=2018-02-10T18%3A07%3A00Z&sp=rl&sv=2017-04-17&sr=b&sig=c628a4fa-7501-4f8a-9f4e-3651fe696e39"
+                          }
+                        }
+                      ],
+                      "id": "75088617-dee2-4657-80de-82bb000c5943",
+                      "createdDateTime": "2019-03-20T14:46:18Z",
+                      "lastActionDateTime": "2019-03-20T14:46:18Z",
+                      "status": "Running",
+                      "locale": "en-US",
+                      "name": "Language model",
+                      "description": "This is a language model"
+                    }
+                  ],
+                  "dataset": {
+                    "dataImportKind": "Acoustic",
+                    "id": "932727fb-015c-493b-9e5d-786f7a4e2c89",
+                    "createdDateTime": "2019-03-20T14:46:18Z",
+                    "lastActionDateTime": "2019-03-20T14:46:18Z",
+                    "status": "Succeeded",
+                    "locale": "en-US",
+                    "name": "Acoustic dataset",
+                    "properties": {
+                      "AcceptedLines": "11",
+                      "RejectedLines": "2",
+                      "ReportUri": "https://www.example.org/AcousticData/ed9b62f0-aa8e-419c-ab55-d7ca83c2f6b3?st=2018-02-09T18%3A07%3A00Z&se=2018-02-10T18%3A07%3A00Z&sp=rl&sv=2017-04-17&sr=b&sig=2b5e9932-8c65-45e5-9f70-9854e1997034"
+                    }
+                  },
+                  "wordErrorRate": 0.0,
+                  "resultsUrl": "https://contoso.com/results/9E32D75F-CEE1-47F5-BA32-65E9BBA18743",
+                  "id": "f480788a-0465-496b-907d-bd447c552edd",
+                  "createdDateTime": "2019-03-20T14:46:18Z",
+                  "lastActionDateTime": "2019-03-20T14:46:18Z",
+                  "status": "Running",
+                  "name": "Accuracy test",
+                  "properties": {}
+                }
+              ]
+            }
+          },
+          "401": {
```

Comment:
please see https://github.com/Azure/azure-rest-api-specs/blob/da93a8f5a90e63c882d176021d6b935671faf0ba/documentation/creating-swagger.md#responses in particular https://github.com/Azure/azure-rest-api-specs/blob/da93a8f5a90e63c882d176021d6b935671faf0ba/documentation/creating-swagger.md#negative-responses
the recommendation is to have a "default" response for error responses which return an error model. If the spec describes each code, AutoRest for SDK code generation will have an "Object" return type for the operations, as "Error" and your actual return type have no other ancestor in common.